src/sql/orderbook/batch_rewards.sql

```diff
@@ -53,32 +180,37 @@ WITH observed_settlements AS (SELECT
                        winning_score,
                        reference_score,
                        -- auction_participation
-                       participating_solvers
+                       participating_solvers,
+                       -- protocol_fees
+                       coalesce(cast(protocol_fee as numeric(78, 0)), 0) as protocol_fee,
+                       coalesce(cast(network_fee_correction as numeric(78, 0)), 0) as network_fee_correction
                      FROM settlement_scores ss
                             -- If there are reported scores,
                             -- there will always be a record of auction participants
                             JOIN auction_participation ap
                                  ON ss.auction_id = ap.auction_id
                        -- outer joins made in order to capture non-existent settlements.
                             LEFT OUTER JOIN observed_settlements os
-                                            ON os.auction_id = ss.auction_id),
+                                            ON os.auction_id = ss.auction_id
+                            LEFT OUTER JOIN batch_protocol_fees bpf
+                                ON bpf.tx_hash = os.tx_hash),
      reward_per_auction as (SELECT tx_hash,
+                                   auction_id,
                                    settlement_block,
                                    block_deadline,
                                    solver,
                                    execution_cost,
                                    surplus,
-                                   fee,
-                                   surplus + fee - reference_score           as uncapped_payment_eth,
-                                   -- Uncapped Reward = CLAMP_[-E, E + exec_cost](uncapped_payment_eth)
-                                   LEAST(GREATEST(-10000000000000000, surplus + fee - reference_score),
-                                         10000000000000000 + execution_cost) as capped_payment,
+                                   protocol_fee + fee - network_fee_correction as fee,-- total fee for ranking
```

Comment:
Should we call this something different? I am a bit confused as two lines later you use a `fee` term in the `uncapped_payment_eth` computation, and it is probably not referring to this total fee

Reply:
We could to `corrected_fee` and later select `corrected_fee as fee` for the uploading. Changing the name for what we upload might be breaking dune queries (but not the queries we use for payments, I guess, which only depend on `capped_payment`).

Another alternative is to repurpose the `fee` field to just be something like network fees. We do not use the field directly in the payment queries, so there it should be fine. Since protocol fees are synced explicitly, we can still recover the corrected quality on dune.

Reply:
The query now computes `protocol_fee` and `network_fee` separately. The payment is thus by `quality = surplus + protocol_fee + network_fee`.

On dune, the entry `network_fee` is still called just `fee` for now to not break existing queries.